source/sass/colors.scss

```diff
@@ -1,3 +1,5 @@
+// [TODO] Old colors to be removed soon
+
```

Comment:
This section will be removed soon. I'm leaving it untouched for now so we can easily compare what's old vs new.

Reply:
Before this gets approved: is there a follow-up filed already? If not, let's file that first.